sql/expression/function/logarithm.go

```diff
@@ -231,15 +231,17 @@ func (l *Log) Eval(
 	}
 
 	// rhs becomes value, lhs becomes base
-	return computeLog(rhs.(float64), lhs.(float64))
+	return computeLog(ctx, rhs.(float64), lhs.(float64))
 }
 
-func computeLog(v float64, base float64) (float64, error) {
+func computeLog(ctx *sql.Context, v float64, base float64) (interface{}, error) {
 	if v <= 0 {
-		return float64(0), ErrInvalidArgumentForLogarithm.New(v)
+		ctx.Warn(3020, ErrInvalidArgumentForLogarithm.New(v).Error())
```

Comment:
is there a list of warning numbers somewhere?

Reply:
This is MySQLs list:
https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html

I don't think we have a list of these in dolt/gms